hosts/azure/jenkins-controller/configuration.nix

```
@@ -445,7 +440,7 @@ in
       RuntimeDirectory = "rclone-http";
       EnvironmentFile = "/var/lib/rclone-jenkins-artifacts/env";
       ExecStart = lib.concatStringsSep " " [
-        "${rclone}/bin/rclone"
+        "rclone"
```

```suggestion
        "${pkgs.rclone}/bin/rclone"
```
rclone is not in PATH for this service. either use from pkgs or add to PATH